src/hex.py

```diff
@@ -6,17 +6,31 @@
 if len(sys.argv) != 3:
     print("Incorrect number of arguments.", file=sys.stderr)
     print(f"Usage: {sys.argv[0]} command string\n", file=sys.stderr)
-    sys.exit(1)
+    # Usage: tells which arguments the program given by sys.argv[0]
+    # requires. 
+    sys.exit(1) # ?
```

Comment:
If the `?` means you are unsure what this is doing, `sys.exit(1)` terminates the script.